qiskit-superstaq/qiskit_superstaq/superstaq_provider.py

```diff
@@ -107,31 +105,8 @@ class SuperstaQProvider(
 
         backend = ss_provider.get_backend('target')
 
-    where `'MY_TOKEN'` is the access token provided by SuperstaQ,
-    and 'target' is the name of the desired backend.
-
-    Args:
-        api_key: A string that allows access to the SuperstaQ API. If no key is provided, then
-            this instance tries to use the environment variable `SUPERSTAQ_API_KEY`. If
-            furthermore that environment variable is not set, then this instance checks for the
-            following files:
-            - `$XDG_DATA_HOME/super.tech/superstaq_api_key`
-            - `$XDG_DATA_HOME/coldquanta/superstaq_api_key`
-            - `~/.super.tech/superstaq_api_key`
-            - `~/.coldquanta/superstaq_api_key`
-            If one of those files exists, then it is treated as a plain text file, and the first
-            line of this file is interpreted as an API key.  Failure to find an API key raises
-            an `EnvironmentError`.
-        remote_host: The location of the API in the form of a URL. If this is None,
-            then this instance will use the environment variable `SUPERSTAQ_REMOTE_HOST`.
-            If that variable is not set, then this uses
-            `https://superstaq.super.tech/{api_version}`,
-            where `{api_version}` is the `api_version` specified below.
-        api_version: Version of the API.
-        max_retry_seconds: The number of seconds to retry calls for. Defaults to one hour.
-        verbose: Whether to print to stdio and stderr on retriable errors.
-    Raises:
-        EnvironmentError: If an API key was not provided and could not be found.
+    where `MY_TOKEN` is the access token provided by SuperstaQ,
```

Comment:
minor nit:
```suggestion
    where `MY_TOKEN` is the access token provided by Superstaq,
```

Reply:
Good catch! Went ahead and switched SuperstaQ -> Superstaq where we could in the docs.